.env

```diff
@@ -17,19 +17,22 @@ HTTPS_PORT=443
 DOCKER_REGISTRY_URL=docker.sdelements.com:443/moodle
 
 # Moodle variables
-# WWWROOT MUST be IP or FQDN
-MOODLE_WWWROOT=https://localhost
-MOODLE_DATAROOT=/opt/moodle/moodledata
 MOODLE_ADMIN_EMAIL=admin@example.com
 MOODLE_ADMIN_PASS=Ch@ngeM3!
+MOODLE_DATAROOT=/opt/moodle/moodledata
```

Comment:
I think DATAROOT and WWWROOT are two separate words (DATA_ROOT, WWW_ROOT) and I think the code readability would improve with underscores.

Reply:
These are the variables in Moodle so I kept them to match:

```php
$CFG->wwwroot   = 'http://34.235.152.190';
$CFG->dataroot  = '/opt/moodledata';
```